### ISSUE TITLE
Minor visual issues

### ISSUE THREAD

**Issue by romansp** [rolandgeider/wger#205]
Hi, @rolandgeider. Prepared list of minor visual issues. Can you review those, and I will later submit PR?

1. Modal dialog 'Share this page' via social media looks kinda wierd. Thinks it needs some rework.
![image](https://cloud.githubusercontent.com/assets/3474842/10254432/3fdc69d8-694c-11e5-81bc-c9e2b93d4ddf.png)

2. 'Get in touch' link in header gets color of background on mouse hover.
![image](https://cloud.githubusercontent.com/assets/3474842/10254455/7acf07ee-694c-11e5-836b-d8cbb80c93ed.png)

3. Navbar action links disappear on small screen if screen is shrinked.
![image](https://cloud.githubusercontent.com/assets/3474842/10254464/93f2f492-694c-11e5-903a-39768dda1bba.png)

4. Consider cancelling margin-top for the first h2 in mid column, so it will be 'in line' with left and right columns.![image](https://cloud.githubusercontent.com/assets/3474842/10255535/b207daae-6953-11e5-93c3-ffc113b961af.png)

5. Set line-height: 1 for those orange buttons, so text will appear in the middle;
![image](https://cloud.githubusercontent.com/assets/3474842/10255623/50dc1fe6-6954-11e5-822c-bb76aa61a7ac.png)

6. Insert space or line-break before [active] label.
![image](https://cloud.githubusercontent.com/assets/3474842/10255697/bad4b91c-6954-11e5-8bef-f7ec007288be.png)

7. The same as previous.
![image](https://cloud.githubusercontent.com/assets/3474842/10255741/13e672ac-6955-11e5-96a7-a3697c5c9607.png)

Will look for some more.

**Comment by romansp** [comment]
Oh, you've just commited font for the missing icons in modal :)

**Comment by rolandgeider** [comment]
nice to see you again @romansp! nice catches!

1. hehe, yeah

2. yes you are right, I could swear I had fixed this, but perhaps I just dreamed it :)

3. this is because the application is not really responsive, for some pages you get a different template if you use a mobile agent. Really fixing this is one thing I want to do in the future but till then, we should fix this

4. definitely

5. this would be the fastest way. The month table has other problems, e.g. when the screen is small (smartphone) and there are many entries in one week, the margins (or paddings?) make the whole table too wide.

6. and 7. yes
